content/docs/csidriver/features/unity.md

```diff
@@ -494,3 +494,126 @@ kubectl edit configmap -n unity unity-config-params
 
 >Note: Prior to CSI Driver for unity version 2.0.0, the log level was allowed to be updated dynamically through `logLevel` attribute in the secret object.
 
+## Tenancy support for Unity NFS
+
+The CSI Unity driver version 2.1.0 and later support Tenancy feature of Unity such that user will be able to associate specific worker nodes (in the cluster) and NFS storage volumes with Tenant.
```

Comment:
Change toi: 

The CSI Unity driver version 2.1.0 (and later versions) support the Tenancy feature of Unity such that the user will be able to associate specific worker nodes (in the cluster) and NFS storage volumes with Tenant.

Reply:
tc